test/extended/etcd/etcd_backup_noconfig.go

```diff
@@ -0,0 +1,386 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	configv1alpha1 "github.com/openshift/api/config/v1alpha1"
+	"github.com/openshift/library-go/test/library"
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/storage/names"
+	"k8s.io/klog/v2"
+	"k8s.io/kube-openapi/pkg/util/sets"
+	"k8s.io/utils/pointer"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+)
+
+const (
+	defaultBackupCRName       = "default"
+	testSchedule              = "*/5 * * * *"
+	testTimeZone              = "UTC"
+	testRetentionType         = configv1alpha1.RetentionTypeNumber
+	testRetentionNumber       = 3
+	backupServerContainerName = "etcd-backup-server"
+	masterNodeLabel           = "node-role.kubernetes.io/master"
+	OpenShiftEtcdNamespace    = "openshift-etcd"
+	backupVolume              = "/var/lib/etcd-auto-backup"
+	backupDirName             = "etcd-auto-backup-dir"
+	podLabelSelector          = "app=etcd"
+
+	// ShellImage allows us to have basic shell tooling, taken from origin:
+	// https://github.com/openshift/origin/blob/6ee9dc56a612a4c886d094571832ed47efa2e831/test/extended/util/image/image.go#L129-L141C2
+	ShellImage = "image-registry.openshift-image-registry.svc:5000/openshift/tools:latest"
```

Comment:
there's a function you can reference here:

> image.ShellImage()

from 	"github.com/openshift/origin/test/extended/util/image"

Reply:
done 👍🏽